index.html

```diff
@@ -83,13 +83,19 @@
 
     </style>
 
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

Comment:
Honestly I don't remember. Although now it's recommended to use https when available.